scripts/update-git-platinum.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Verify that the script is run from project root.
+BASE=$(basename $(pwd))
+
+if [ "${BASE}" != "radicle-surf" ]
+then
+   echo "this script should be run from the root of radicle-surf"
```

Comment:
```suggestion
   echo "ERROR: this script should be run from the root of radicle-surf"
```

Just to make it clear :smile: